app/src/main/java/com/github/libretube/repo/LocalFeedRepository.kt

```diff
@@ -59,6 +61,7 @@ class LocalFeedRepository : FeedRepository {
     }
 
     private suspend fun refreshFeed(channelIds: List<String>, minimumDateMillis: Long) {
+        val extractionCount = AtomicInteger()
```

Comment:
We don't actually need an atomic integer here, the outer for loop runs only once at a time.